File: BankBankApp.DAL/BillerDAL.cs
using System;
using System.Collections.Generic;
using System.Text;
using BankBankApp.BO;
using BankBankApp.DAL.Interfaces;

namespace BankBankApp.DAL
{
    public class BillerDAL : IBiller
    {
        public void Create(Biller obj)
        {
            throw new NotImplementedException();
        }

        public void Delete(int id)
        {
            throw new NotImplementedException();
        }

        public IEnumerable<Biller> Get()
        {
            throw new NotImplementedException();
        }

        public Biller GetByBillerAccountNumber(string billerAccountNumber)
        {
            throw new NotImplementedException();
        }

        public Biller GetByID(int id)
        {
            throw new NotImplementedException();
        }

        public Biller Update(Biller obj, int id)
        {
            throw new NotImplementedException();
        }
    }
}

File: BankBankApp.DAL.vb/BillerDAL.vb
Imports BankBankApp.BO
Imports BankBankApp.Interfaces

Public Class BillerDAL
    Implements IBiller

    Public Function Create(entity As Biller) As Integer Implements ICrud(Of Biller).Insert
        Throw New NotImplementedException()
    End Function

    Public Function Delete(id As Integer) As Integer Implements ICrud(Of Biller).Delete
        Throw New NotImplementedException()
    End Function

    Public Function GetAll() As List(Of Biller) Implements ICrud(Of Biller).GetAll
        Throw New NotImplementedException()
    End Function

    Public Function GetById(id As Integer) As Biller Implements ICrud(Of Biller).GetById
        Throw New NotImplementedException()
    End Function

    Public Function Update(entity As Biller) As Integer Implements ICrud(Of Biller).Update
        Throw New NotImplementedException()
    End Function

End Class
